src/renderer/views/stake/Share/ShareView.tsx

```diff
@@ -40,7 +46,11 @@ export const ShareView: React.FC<{ asset: Asset }> = ({ asset }) => {
             target={asset.chain}
             poolShare={helpers.getPoolShare(stake, pool)}
             assetStakedShare={assetStakedShare}
-            basePriceAsset={priceSymbol}
+            // TODO (@thatThorchainGuy) price symobl as `basePriceAsset` - is that correct?
```

Comment:
yep. thats correct. agree thats this is a little bit confusing name. previously there was an asset at the `PoolCard` component but was replaced to the asset's symbol itself which is calculated at the midgard service as a single source of thruth